server/src/services/cleanUserSubmission.js

```diff
@@ -0,0 +1,18 @@
+const cleanUserSubmission = formInput => {
+  for(const box in formInput){
+    let adjustedBox = formInput[box]
+    
+    if(formInput[box].length < 9){
+      for(let x = 0; x< 9; x++){
+        formInput[box][x] = 0
+      }
+    }
```

Comment:
I think I've lost the thread so maybe Im missing something with this. If the elements in a box are less than 9....change all elements to 0?